ts/components/messages/MessageListComponent.tsx

```diff
@@ -71,7 +71,6 @@ class MessageListComponent extends React.Component<Props> {
     const refreshControl = (
       <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
     );
-
```

Comment:
why this change?

Reply:
Sorry, i think i removed the line during implementing an other const that i removed later